paragon/blocks/_authn.scss

```diff
@@ -133,6 +135,131 @@
 
             h1.data-hj-suppress {
                 max-width: none;
+                color: $white;
+            }
+        }
+
+        .pp-page-content {
+            @include media-breakpoint-down(xs) {
+                margin: 1.5rem .5rem !important;
+            }
+        }
+
+        h1.welcome-to-platform {
+            color: $white;
+            margin-bottom: 40px;
+        }
+
+        .organization-select-info {
+            color: #888;
+            font-size: 14px;
+        }
+
+        .authn-organization-form {
+            line-height: normal;
+
+            .pgn__form-control-decorator-group {
+                margin-right: 0 !important;
+            }
+
+            .pgn__form-label {
+                margin-bottom: 1rem;
+                line-height: 1.4;
+                font-size: 15px;
+            }
+
+            #is_organization_registered .pgn__form-label {
+                @media screen and (max-width: 450px) {
+                    max-width: 280px;
+                }
+            }
+
+            #react-select-2-option-0 {
+                line-height: 1.1;
+            }
+        }
+
+        .authn-tooltip-icon {
+            color: $white;
+            display: inline-block;
+            vertical-align: top;
+            margin-left: 7px;
+            background: $authn-tooltip-bg;
+            text-align: center;
+            font-weight: 700;
+            font-size: 12px;
+            position: relative;
+            top: 3px;
+            width: 16px;
+            height: 16px;
+            line-height: 16px;
+            border-radius: 50%;
+            cursor: pointer;
+
+            &:hover .authn-tooltip {
+                display: block;
+            }
+
+            .authn-tooltip {
+                display: none;
+                position: absolute;
+                color: $white;
+                font-weight: 400 !important;
+                background: $authn-tooltip-bg;
+                z-index: 1;
+                font-size: 13px;
+                line-height: 1.5;
+                padding: 10px 15px;
+                border-radius: 4px;
+                width: 175px;
+
+                &:before {
+                    content: "";
+                    position: absolute;
+                    width: 0;
+                    height: 0;
+                    border-style: solid;
+                }
+
+                @media screen and (min-width: 611px) {
+                    left: calc(100% + 15px);
+                    top: 50%;
+                    transform: translateY(-50%);
+
+                    &:before {
+                        left: -12px;
+                        top: 50%;
+                        transform: translateY(-50%);
+                        border-width: 7.5px 12px 7.5px 0;
```

Comment:
[important] Using floating point numbers in CSS may render differently in browsers due to different styling algorithms